external_auth/views.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.shortcuts import render
+
+from django.contrib import messages
+from external_auth.forms import PakwheelsLoginForm
+from external_auth.pakwheels_scraper import PakwheelsScraper
+from external_auth.pakwheels_profile import PakwheelsProfile
+
+
+def pakwheels_registration(request):
+
+    if request.method == 'POST':
+        form = PakwheelsLoginForm(request.POST)
+        username = form.data['username']
+        password = form.data['password']
+        pakwheels_scraper = PakwheelsScraper(
+            username,
+            password,
+        )
+        pakwheels_profile_response = pakwheels_scraper.login_and_retrieve_profile()
+
+        if pakwheels_profile_response:
+            messages.success(
+                request, 'Credentials are correct.'
+                'Your profile is importing from Pakwheels. '
+                'Use Pakwheels username and password to login.'
+            )
+            pakwheels_profile = PakwheelsProfile(
+                first_name=pakwheels_profile_response['first_name'],
+                last_name=pakwheels_profile_response['last_name'],
+                gender=pakwheels_profile_response['user_gender'],
+                user_name=pakwheels_profile_response['user_name'],
+                password=password,
+                email=pakwheels_profile_response['user_email'],
+                birthday=pakwheels_profile_response['user_birthday'],
+                city=pakwheels_profile_response['user_city'],
+                country=pakwheels_profile_response['user_country'],
+            )
+
+            new_user, created = User.objects.get_or_create(
```

Comment:
how would you know its a new user before checking the `created` value? :)
`user, is_created = User.objects.get_or_create(`